Demos/Blazorise.Demo/Pages/Tests/ChartsPage.razor.cs

```diff
@@ -19,14 +19,11 @@ public partial class ChartsPage
         {
             Scales = new()
             {
-                YAxes = new()
+                Y = new()
                 {
-                    new()
+                    Ticks = new AxisTicks
                     {
-                        Ticks = new AxisTicks
-                        {
-                            Callback = ( value, index, values ) => $"{value / 1000} K"
-                        }
+                        Callback = ( value, index, values ) => value / 1000 + "K"
```

Comment:
Actually the previous way explicits tells .NET it's a string in there, right?
This way calls in an operator with the int value boxed, probably more costly? These things are still neglible eitherway.. :P
![image](https://user-images.githubusercontent.com/22283161/143504837-c7af0a1e-d1af-4293-b1d3-991942b9da95.png)

Reply:
This will be compiler and run in JS, never in C#. That's why an expression needs to be as simple a possible.